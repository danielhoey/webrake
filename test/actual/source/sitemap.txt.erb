<% root_url = "http://www.testsite.com" %>
<%= @source_files.map {|f| "#{root_url}/#{f.path}"}.sort.reverse.join("\n") %>

