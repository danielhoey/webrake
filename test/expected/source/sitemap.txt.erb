../../actual/source/sitemap.txt.erb