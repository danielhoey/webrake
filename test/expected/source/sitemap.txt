http://www.testsite.com/page1.html
http://www.testsite.com/index.html
http://www.testsite.com/blog/post1.html
